coderd/httpmw/userparam.go

```diff
@@ -36,15 +36,15 @@ func UserParam(r *http.Request) database.User {
 // parameter.
 //
 //nolint:revive
```

Comment:
Is the `//nolint` here still necessary?

Reply:
Oh, I guess that boolean param is probably why it was complaining.